src/components/LexicalCategoryInput.vue

```diff
@@ -19,6 +21,18 @@ const searchForItems = searcher.searchItems.bind( searcher );
 const config = useConfig();
 const lexicalCategorySuggestions = config.lexicalCategorySuggestions;
 const exampleLexCategory = config.placeholderExampleData.lexicalCategoryLabel;
+const store = useStore();
+const error = computed( () => {
+	if ( !store.state.perFieldErrors.lexicalCategoryErrors.length ) {
+		return null;
+	}
+	return {
+		type: 'error' as const,
+		message: messages.getUnescaped(
+			store.state.perFieldErrors.lexicalCategoryErrors[ 0 ].messageKey,
+		),
+	};
```

Comment:
I still think this code should be extracted into a function somewhere, taking the `PerFieldError[]` and returning an object `{ type, message }`. I guess it would be another plugin, so that it can access the `messages` without it being awkward? (Or take those as another parameter.)

Reply:
mh, maybe let's do this in an extra pull request at the end